marklogic-data-hub/src/main/java/com/marklogic/hub/EntityManager.java

```diff
@@ -75,6 +81,8 @@
 
     List<HubEntity> getEntities();
 
+    List<JsonNode> getEntitiesJsonNode();
```

Comment:
Let's not add a method to this interface. We get almost no value out of the interfaces, except that we imply that the interface will stay constant for someone that wants to use the DHF Java API directly - but we provide no support for that other than publishing the javadocs. Just add the method to EntityManagerImpl and call that directly in the migration class.